### PR TITLE
Set up cloud CI

### PR DESCRIPTION
**Pull request by yuriidanyliak**
Look at circle CI.

### CODE REVIEW

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, set up the [Circle CI](https://circleci.com/gh/yuriidanyliak/TaDa), integrated with Github and [Heroku](https://immense-peak-9095.herokuapp.com/) (provides automatic deployment when pushing to git).

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, That is cool! :+1:

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+deployment:
+  staging:
+    branch: set-up-cloud-ci
```
Comment (sergeyradkevich) [inline circle.yml]: Hey @yuriidanyliak, is this the branch in the repo for auto deploy based on a push?

Comment (sergeyradkevich) [inline circle.yml]: If yes, then I believe that we need to stick to `master` or create `staging` branch for CI process.

Comment (yuriidanyliak) [inline circle.yml]: @sergeyradkevich, yes. As I understood, for every branch/pull request should be its own version of circle.yml (in current case, with its own value of `branch` property).

**Comment by sergeyradkevich** [comment]
Hm, that would not usable though. Will discuss today.

File: circle.yml (hunk 1 above)
Comment (yuriidanyliak) [inline circle.yml]: On another hand, I have configured Heroku to build automatically when pushing to `master` branch. Also there is an ability to automatically create new apps and build corresponding app for every new pull request, as I understood.

Comment (yuriidanyliak) [inline circle.yml]: So, after a merging of this pull request to master branch all we need to do for every new branch - simply change one line in `circle.yml` to current branch name.

Comment (yuriidanyliak) [inline circle.yml]: Also, I think we can use Circle CI environment variables or git variables to detect current git branch and replace hardcoded branch name in configurations to one which got from the environment.

**Comment by yuriidanyliak** [comment]
[Disabled](http://img.in6k.com/screens/screens0d41a0fc8784.png) automatic deployment in Heroku web interface.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, updated configuration as was mentioned earlier. Now current Heroku instance is using only for master branch deploying.

**Pull request merged**